Tribler/Test/test_gui_dialogs.py

```diff
@@ -96,7 +96,7 @@ def do_save_dialog():
             self.assert_(isinstance(dialog, AddTorrent), 'could not find AddTorrent')
 
             self.callLater(1, lambda: do_assert(dialog))
-            dialog.magnet.SetValue(r'http://torrent.fedoraproject.org/torrents/Fedora-20-i386-DVD.torrent')
```

Comment:
Use the constants defined in Tribler/Test/test_libtorrent_download.py for torrent paths, so we can change it easily if we need to.